parmed/amber/parameters.py

```diff
@@ -211,7 +211,7 @@ def __init__(self, *filenames):
     #===================================================
 
     @classmethod
-    def from_leaprc(cls, fname):
+    def from_leaprc(cls, fname, addAtomTypes=True):
```

Comment:
Except for areas where the ParmEd and OpenMM APIs overlap (like `createSystem` and the like), ParmEd uses the `snake_naming_convention`.